syncstorage/tests/functional/test_storage.py

```diff
@@ -272,6 +275,7 @@ def test_get_collection(self):
         for index, sortindex in (('0', -1), ('1', 34), ('2', 12)):
             bso = {'id': index, 'payload': 'x', 'sortindex': sortindex}
             self.app.post_json(self.root + '/storage/col2', [bso])
+            time.sleep(0.01)
```

Comment:
Likewise, each independent request here should get an increased timestamp without the client having to worry about it.